docs/examples/ODE_example.py

```diff
@@ -0,0 +1,367 @@
+
+# Generate all the ground truths detections, we set the noise to 0 in this case,
+# please note the timesteps
+for k in range(1, simulation_step + 1):
+    truth.append(GroundTruthState(
+        transition_model.function(truth[k - 1],
+                                  noise=False,
+                                  time_interval=time_interval),
+        timestamp=start_time + timedelta(seconds=k*time_interval.total_seconds())))
+
+# Prepare the measurement model and true detections
+from stonesoup.models.measurement.nonlinear import CartesianToElevationBearingRange
+from stonesoup.types.detection import TrueDetection
+
+# define the measurement model
+measurement_model = CartesianToElevationBearingRange(
+    ndim_state=6,
+    mapping=(0, 2, 4),
+    noise_covar=np.diag([np.radians(0.005), np.radians(0.005), 100])
+)
+
+# Create the measurements
+measurements = []
+for state in truth:  # loop over the ground truths
+    measurement = measurement_model.function(state=state, noise=True)
+    measurements.append(TrueDetection(
+        state_vector=measurement,
+        timestamp=state.timestamp,
+        groundtruth_path=truth,
+        measurement_model=measurement_model))
+
+# %%
+# Let's visualise the results. We add the center of Earth as a
+# fixed platform.
+
+from stonesoup.platform.base import FixedPlatform
+
+Center_earth = FixedPlatform(
+    states=GaussianState(
+        np.array([0, 0, 0, 0, 0, 0]),
+        np.diag([1, 1, 1, 1, 1, 1])),
+    position_mapping=(0, 2, 4))
+
+from stonesoup.plotter import Plotter, Dimension
+
+plotter = Plotter(Dimension.THREE)
+plotter.plot_ground_truths(truth, [0, 2, 4])
+plotter.plot_measurements(measurements, [0, 2, 4])
+plotter.plot_sensors(Center_earth, mapping=[0, 1, 2], sensor_label='Center Earth')
+plotter.fig
+
+# %%
+# 3. Instantiate the tracker components and run the tracker
+# ---------------------------------------------------------
+# We employ a :class:`~.ExtendedKalmanUpdater` and
+# :class:`~.ExtendedKalmanPredictor` to perform the tracking,
+# since we are dealing with a single target tracker we don't specify
+# any track deleter and initiator. As well, in this simpler case
+# we don't consider any misdetection or clutter.
+
+from stonesoup.predictor.kalman import ExtendedKalmanPredictor
+from stonesoup.updater.kalman import ExtendedKalmanUpdater
+from stonesoup.types.track import Track
+from stonesoup.types.hypothesis import SingleHypothesis
+
+predictor = ExtendedKalmanPredictor(transition_model)
+updater = ExtendedKalmanUpdater(measurement_model)
+
+# Prepare the tracking
+track = Track([prior])
+for measurement in measurements:
+    prediction = predictor.predict(prior, timestamp=measurement.timestamp)
+    hypothesis = SingleHypothesis(prediction, measurement)
+    post = updater.update(hypothesis)
+    track.append(post)
+    prior = track[-1]
+
+# Now visualise the results
+plotter.plot_tracks(track, [0, 2, 4], track_label='Track')
+plotter.fig
```

Comment:
The results look like track following very closely to the measurements. Is this what you'd expect? An issue calculating Q??

![image](https://github.com/dstl/Stone-Soup/assets/19266036/cf96c680-5b93-4b97-acd0-b94746b58e14)